src/client/VZSidebar/index.tsx

```diff
@@ -199,11 +247,63 @@ export const VZSidebar = ({
             </div>
           </div>
         ) : filesExist ? (
+
+          <List
+            aria-label="Example List"
+            selectionMode="multiple"
+            selectionBehavior="replace"
+            onAction={(key) => handleFileDoubleClick(key)}
```

Comment:
I think this one can be simplified.
```suggestion
            onAction={handleFileDoubleClick}
```